src/lib/I18n.js

```diff
@@ -8,54 +7,55 @@ class I18n {
         this._dict = dictionary;
 
         if (!(fallbackLanguage in this._dict)) {
-            throw new Error(`Fallback language "${ fallbackLanguage }" not defined`);
+            throw new Error(`Fallback language "${fallbackLanguage}" not defined`);
         }
         /** @type {string} */
         this._fallbackLanguage = fallbackLanguage;
 
         /** @type {string} */
-        this._language;
+        this._language = '';
```

Comment:
unnecessary init. Just doing that to avoid the linter to complain? We shouldn't "bend" the code to make the linter shut up.